improver_tests/psychrometric_calculations/cloud_condensation_level/test_CloudCondensationLevel.py

```diff
@@ -101,10 +101,9 @@ def metadata_ok(ccl: Tuple[Cube, Cube], baseline: Cube, model_id_attr=None) -> N
     t_at_ccl, p_at_ccl = ccl
     assert t_at_ccl.long_name == "air_temperature_at_condensation_level"
     assert t_at_ccl.units == "K"
-    assert "height" not in [c.name() for c in t_at_ccl.coords()]
+    assert "height" not in [c.name() for c in t_at_ccl.coords() + p_at_ccl.coords()]
```

Comment:
This is not quite what I had in mind. You loop over `t_at_ccl` and `p_at_ccl` below using `for cube in ccl`. You can just insert the test there (where you do lots of other coordinate checking).

`assert "height" not in [c.name() for c in cube.coords()]`

Reply:
done